server/package.json

```diff
@@ -15,6 +15,8 @@
   "author": "wellcomecollection.dev@wellcome.ac.uk",
   "license": "MIT",
   "dependencies": {
+    "@frctl/fractal": "^1.0.8",
+    "@frctl/nunjucks": "https://github.com/gestchild/nunjucks/tarball/66b327d3c9ac24f97784332c09669336fe4c650a",
```

Comment:
Using a fork for now as the current nunjucks adapter for fractal doesn't allow full paths for includes, which is what we're using in the app. Hoping to get the change into the main repo then can switch back